### PR TITLE
Hotfix/243

### LINKED ISSUE #243: HR: keine neuen Sendungen mehr
Wie im [Forum](https://forum.mediathekview.de/topic/723/hr-sendungen-nur-bis-15-sep/) gemeldet, findet der HR-Crawler seit Mitte September keine neuen Sendungen mehr.

Erste Analyse: die initial geladene Seite wird auf die Startseite der HR-Mediathek umgeleitet, offensichtlich existiert die URL nicht mehr.

### PR DESCRIPTION
**Pull request by pidoubleyou**
Fixes #243

### CODE REVIEW

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungDeserializerTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungDeserializerTest.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungDeserializerTest.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class HrSendungDeserializerTest {
+    
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][] {
```
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungDeserializerTest.java]: MINOR Remove this array creation and simply pass the elements. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializerTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializerTest.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializerTest.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import static org.junit.Assert.*;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class HrSendungOverviewDeserializerTest {
+    
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][] {
```
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializerTest.java]: MINOR Remove this array creation and simply pass the elements. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)

File: src/main/java/mServer/crawler/sender/hr/HrSendungenDto.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungenDto.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/test/developTest/java/mServer/test/HtmlFileReader.java [hunk 6]
```diff
@@ -0,0 +1,24 @@
+package mServer.test;
```
Comment (alex1702) [inline src/test/developTest/java/mServer/test/HtmlFileReader.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungenListDeserializer.java [hunk 7]
```diff
@@ -0,0 +1,67 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungenListDeserializer.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungenListDeserializerTest.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungenListDeserializerTest.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializer.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializer.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/MediathekHr.java [hunk 10]
```diff
@@ -19,26 +19,35 @@
...
+import org.jsoup.nodes.Document;
 
 public class MediathekHr extends MediathekReader {
 
     public final static String SENDERNAME = Const.HR;
-    private MSStringBuilder seite = new MSStringBuilder(Const.STRING_BUFFER_START_BUFFER);
-    private MSStringBuilder rubrikSeite = new MSStringBuilder(Const.STRING_BUFFER_START_BUFFER);
-
+    
+    private final static String URL_SENDUNGEN = "http://www.hr-fernsehen.de/sendungen-a-z/index.html";
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/MediathekHr.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/main/java/mServer/crawler/sender/MediathekHr.java [hunk 11]
```diff
@@ -19,26 +19,35 @@
  */
 package mServer.crawler.sender;
 
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-
-import de.mediathekview.mlib.Config;
 import de.mediathekview.mlib.Const;
 import de.mediathekview.mlib.daten.DatenFilm;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/MediathekHr.java]: MINOR Remove this unused import 'de.mediathekview.mlib.daten.DatenFilm'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/mServer/crawler/sender/MediathekHr.java [hunk 12]
```diff
@@ -19,26 +19,35 @@
...
-import de.mediathekview.mlib.tool.Functions;
+import de.mediathekview.mlib.daten.ListeFilme;
 import de.mediathekview.mlib.tool.Log;
-import de.mediathekview.mlib.tool.MSStringBuilder;
-import mServer.crawler.CrawlerTool;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/MediathekHr.java]: MINOR Remove this unused import 'java.util.concurrent.CopyOnWriteArrayList'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/mServer/crawler/sender/hr/HrSendungDeserializer.java [hunk 13]
```diff
@@ -0,0 +1,155 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungDeserializer.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungDeserializer.java [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+
+    private final DateTimeFormatter dateFormatHtml = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mmZ");
+    private final DateTimeFormatter dateFormatDatenFilm = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private final DateTimeFormatter timeFormatDatenFilm = DateTimeFormatter.ofPattern("HH:mm:ss");
+
+    private static final Logger LOG = LogManager.getLogger(HrSendungDeserializer.class);
+    
+    public DatenFilm deserialize(String theme, String documentUrl, Document document) {
+        
+        String date = "", description, time = "", title, videoUrl;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungDeserializer.java]: MINOR Declare "description" on a separate line. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1659)
MINOR Declare "time" on a separate line. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1659)
MINOR Declare "title" on a separate line. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1659)
MINOR Declare "videoUrl" on a separate line. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1659)

File: src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java [hunk 15]
```diff
@@ -0,0 +1,58 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+    @Override
+    public ListeFilme call() throws Exception {
+        ListeFilme list = new ListeFilme();
+        try {
+            if (!Config.getStop()) {
+                Document overviewDocument = Jsoup.connect(dto.getUrl()).get();
+                List<String> detailUrls = overviewDeserializer.deserialize(overviewDocument);
+                detailUrls.forEach(detailUrl -> {
+                    if (!Config.getStop()) {
+                        try {
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java]: MAJOR Extract this nested try block into a separate method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)

File: src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+    private final HrSendungenDto dto;
+    private final HrSendungOverviewDeserializer overviewDeserializer = new HrSendungOverviewDeserializer();
+    private final HrSendungDeserializer sendungDeserializer = new HrSendungDeserializer();
+    
+    public HrSendungOverviewCallable(HrSendungenDto aDto) {
+        dto = aDto;
+    }
+    
+    @Override
+    public ListeFilme call() throws Exception {
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java]: CRITICAL Refactor this method to reduce its Cognitive Complexity from 18 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)

**Comment by alex1702** [comment]
SonarQube analysis reported 23 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 18 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [hr_sendung_hessenschau_detail.html](https://github.com/mediathekview/MServer/blob/c5f7ec2407fa2a6453932a83ae9eca5235f4f701/src/test/developTest/resources/hr/hr_sendung_hessenschau_detail.html): Current file has 1004 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)
1. ![MAJOR][MAJOR] [hr_sendung_hessenschau_list.html](https://github.com/mediathekview/MServer/blob/c5f7ec2407fa2a6453932a83ae9eca5235f4f701/src/test/developTest/resources/hr/hr_sendung_hessenschau_list.html): Current file has 1173 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)
1. ![MAJOR][MAJOR] [hr_sendungen.html](https://github.com/mediathekview/MServer/blob/c5f7ec2407fa2a6453932a83ae9eca5235f4f701/src/test/developTest/resources/hr/hr_sendungen.html): Current file has 1392 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by alex1702** [comment]
SonarQube analysis reported 14 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 11 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [hr_sendung_hessenschau_detail.html](https://github.com/mediathekview/MServer/blob/16edc2fd093a832a724aa614b15bb136970b1e27/src/test/developTest/resources/hr/hr_sendung_hessenschau_detail.html): Current file has 1004 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)
1. ![MAJOR][MAJOR] [hr_sendung_hessenschau_list.html](https://github.com/mediathekview/MServer/blob/16edc2fd093a832a724aa614b15bb136970b1e27/src/test/developTest/resources/hr/hr_sendung_hessenschau_list.html): Current file has 1173 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)
1. ![MAJOR][MAJOR] [hr_sendungen.html](https://github.com/mediathekview/MServer/blob/16edc2fd093a832a724aa614b15bb136970b1e27/src/test/developTest/resources/hr/hr_sendungen.html): Current file has 1392 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungDeserializerTest.java [hunk 18]
```diff
@@ -0,0 +1,103 @@
+package de.mediathekview.mserver.crawler.sender.hr;
```
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungDeserializerTest.java]: CRITICAL This file "HrSendungDeserializerTest.java" should be located in "de/mediathekview/mserver/crawler/sender/hr" directory, not in "/home/travis/build/mediathekview/MServer/src/test/developTest/java/mServer/crawler/sender/hr". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializerTest.java [hunk 19]
```diff
@@ -0,0 +1,48 @@
+package de.mediathekview.mserver.crawler.sender.hr;
```
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializerTest.java]: CRITICAL This file "HrSendungOverviewDeserializerTest.java" should be located in "de/mediathekview/mserver/crawler/sender/hr" directory, not in "/home/travis/build/mediathekview/MServer/src/test/developTest/java/mServer/crawler/sender/hr". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)

File: src/main/java/mServer/crawler/sender/hr/HrSendungenDto.java [hunk 20]
```diff
@@ -0,0 +1,23 @@
+package de.mediathekview.mserver.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungenDto.java]: CRITICAL This file "HrSendungenDto.java" should be located in "de/mediathekview/mserver/crawler/sender/hr" directory, not in "/home/travis/build/mediathekview/MServer/src/main/java/mServer/crawler/sender/hr". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)

File: src/main/java/mServer/crawler/sender/hr/HrSendungDeserializer.java [hunk 21]
```diff
@@ -0,0 +1,159 @@
+package de.mediathekview.mserver.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungDeserializer.java]: CRITICAL This file "HrSendungDeserializer.java" should be located in "de/mediathekview/mserver/crawler/sender/hr" directory, not in "/home/travis/build/mediathekview/MServer/src/main/java/mServer/crawler/sender/hr". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)

File: src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java [hunk 22]
```diff
@@ -0,0 +1,70 @@
+package de.mediathekview.mserver.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java]: CRITICAL This file "HrSendungOverviewCallable.java" should be located in "de/mediathekview/mserver/crawler/sender/hr" directory, not in "/home/travis/build/mediathekview/MServer/src/main/java/mServer/crawler/sender/hr". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)

**Comment by alex1702** [comment]
SonarQube analysis reported 14 issues
* ![CRITICAL][CRITICAL] 8 critical
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [hr_sendung_hessenschau_detail.html](https://github.com/mediathekview/MServer/blob/7609aaa2eca249df12e6a73977f85c9e7a170000/src/test/developTest/resources/hr/hr_sendung_hessenschau_detail.html): Current file has 1004 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)
1. ![MAJOR][MAJOR] [hr_sendung_hessenschau_list.html](https://github.com/mediathekview/MServer/blob/7609aaa2eca249df12e6a73977f85c9e7a170000/src/test/developTest/resources/hr/hr_sendung_hessenschau_list.html): Current file has 1173 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)
1. ![MAJOR][MAJOR] [hr_sendungen.html](https://github.com/mediathekview/MServer/blob/7609aaa2eca249df12e6a73977f85c9e7a170000/src/test/developTest/resources/hr/hr_sendungen.html): Current file has 1392 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungDeserializerTest.java (hunk 1 above)
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungDeserializerTest.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializerTest.java (hunk 3 above)
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializerTest.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungenDto.java (hunk 5 above)
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungenDto.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungenListDeserializer.java (hunk 7 above)
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungenListDeserializer.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/test/developTest/java/mServer/crawler/sender/hr/HrSendungenListDeserializerTest.java (hunk 8 above)
Comment (alex1702) [inline src/test/developTest/java/mServer/crawler/sender/hr/HrSendungenListDeserializerTest.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializer.java (hunk 9 above)
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungOverviewDeserializer.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungDeserializer.java [hunk 23]
```diff
@@ -0,0 +1,159 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungDeserializer.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

File: src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java [hunk 24]
```diff
@@ -0,0 +1,70 @@
+package mServer.crawler.sender.hr;
```
Comment (alex1702) [inline src/main/java/mServer/crawler/sender/hr/HrSendungOverviewCallable.java]: MINOR Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

**Comment by alex1702** [comment]
SonarQube analysis reported 14 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 11 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [hr_sendung_hessenschau_detail.html](https://github.com/mediathekview/MServer/blob/566be028cde938058fe3b6f93bec6eecfbcca2e6/src/test/developTest/resources/hr/hr_sendung_hessenschau_detail.html): Current file has 1004 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)
1. ![MAJOR][MAJOR] [hr_sendung_hessenschau_list.html](https://github.com/mediathekview/MServer/blob/566be028cde938058fe3b6f93bec6eecfbcca2e6/src/test/developTest/resources/hr/hr_sendung_hessenschau_list.html): Current file has 1173 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)
1. ![MAJOR][MAJOR] [hr_sendungen.html](https://github.com/mediathekview/MServer/blob/566be028cde938058fe3b6f93bec6eecfbcca2e6/src/test/developTest/resources/hr/hr_sendungen.html): Current file has 1392 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFileLengthCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by pidoubleyou** [comment]
die HTML-Seiten sind Beispielseiten für UnitTests, somit müssen die major Issues ignoriert werden.

**Pull request merged**